src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const personPartner = person.sex === 'm'
+    ? `${person.partnerName} is my wife`
+    : `${person.partnerName} is my husband`;
```

Comment:
those two lines almost similar, to not repeat it: 
```suggestion
  const personPartner = person.sex === 'm'
    ? ` wife`
    : `husband`;
  const maritalStatus =  person.isMaried 
    ? `${person.partnerName} is my ${personPartner}`
    : 'I am not married'
```